src/renderingManager.js

```diff
@@ -39,27 +39,6 @@ export function newRenderingManager(win, environment) {
     } else {
       renderLegacy(doc, targetingData.adId);
     }
-    
-    // check for winurl and replace BIDID token with value if it exists
-    if (targetingData.winurl && targetingData.winbidid) {
```

Comment:
Is this logic not needed anymore ? @SyntaxNode @hhhjort Is this used anywhere in Perbid server ?

Reply:
This looks like this is firing off a win notification if a win URL is supplied? PBS doesn't touch win notifications, it is not in the loop to know if a bid wins the final auction. So I cannot say if overall if this is needed, we have no metrics on how often such a pixel may be supplied with a bid.

Reply:
I made the assumption that PBS-Go hasn't started implementing events yet. If that's true, then there's no backwards compatibility issue. This is essentially changing the design we had done in PBS-Java because it ran into problem in the real world.